src/controller/AutomatedSolver.java

```diff
@@ -1,60 +0,0 @@
-package controller;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Queue;
-
-import model.Labyrinth;
-import model.Position;
-
-public class AutomatedSolver implements LabyrinthSolver {
-	private Labyrinth l;
-
-	@Override
-	public Labyrinth getLabyrinth() {
-		return l;
-	}
-
-	@Override
-	public void setLabyrinth(Labyrinth l) {
-		this.l = l;
-	}
-
-	public void solve() {
-		Position current = new Position(l.getStartCell());
```

Comment:
If the lab's validity is not "forced" at creation, every usage of it should be verified :D. 
(I keep pushing the RAII :). These are all guidelines, take what you wish ;) )